flashlight/memory/MemoryManagerAdapter.cpp

```diff
@@ -66,6 +66,10 @@ af_memory_manager MemoryManagerAdapter::getHandle() const {
   return interface_;
 }
 
+void MemoryManagerAdapter::setOption(unsigned key, const std::string& value) {
```

Comment:
I think we should pursue an approach where we put specific methods on memory manager implementations that support them rather than having string options. Doing things at compile time seems like it'll make things easier to reason about.

Reply:
done